docs/Installation Guide.md

```diff
@@ -0,0 +1,319 @@
+
+To view the sql commands that will be generated from `syncdb`, run the command:
+
+    python manage.py sqlall app_name_here
+
+To generate the database tables that correspond to the Django models, run the command:
+
+    python manage.py syncdb
+
+After running the `syncdb` command, you should get the following prompt. Type in `yes`:
+
+    You just installed Djano's auth system, which means you don't have any superusers defined.
+    Would you like to create one now? (yes/no): yes
+
+Create a superuser with the following credentials (for now):
+```
+Username: admin
+Email: your email
+Password: mypassword
+```
+Check that the tables were created by starting the postgres client and viewing the tables using the `\dt` command.
+```
+psql -U myuser -d webapp -h localhost -W
+```
+```
+\dt
+```
+Make sure to exit the postgres client before proceeding to the next steps:
+
+    \q
+
+## Run Development Server
+
+Change directory to where you can find the **manage.py** file (this is located in the top level directory for the project).
+
+Start the development server by running the command (this runs the development server on the VM):
+
+    python manage.py runserver [::]:8000
+
+
+## Try out Mobile App Control Center
+
+You can now try out the project by going to [http://localhost:8001](http://localhost:8001) on a browser on your local machine.
+
+Try logging in with your superuser credentials. After clicking on `Submit` you will get the following message:
+
+    No Pcuser associated! Add a pcuser from admin
+
+We need to create a pcuser that is associated with the superuser we created previously. To do this, go to the Django admin panel at [http://localhost:8001/admin](http://localhost:8001/admin) on your browser.
+
+On the Django admin login page, login with your superuser credentials.
+
+Once you have logged in, click on `Add` beside `Pcusers`.
+
+In the form, enter in these fields:
+```
+User: admin
+Location: Your location
+Phone: Your phone number
+Gender: Your gender
+Reset pass: 1 (not sure what this field does)
+Imageobj: Pick any random image from your computer
+Verified: 1 (not sure what this field does)
+```
+Click on `Save` and logout.
+
+Go back to [http://localhost:8001](http://localhost:8001)
+
+Login again with your superuser credentials, it should work since we have associated our superuser with a pcuser. You are now free to try out the application.
+
+## Exit the Virtual Machine
+
+Once you are done with testing out and running the project, you may want to exit the VM and suspend or shut it down by running these commands:
+
+Exit out of the ssh session with the VM by running:
+
+    exit
+
+To put the VM in suspend mode, run the command:
+
+    vagrant suspend
+
+Alternatively, to shut down the VM, run the command:
+
```

Comment:
If possible I would prefer to reduce the manual intervention needed for installation ( thats a big barrier specially for non-tech folks and also leaves room for things to go wrong) .. Ideally user should be able to to vagrant up and thats it! .. we can try to include all the dependencies, installations , service starts , creating DB e.t.c in a script and then call the script from vagrantfile. That ways we also make it reproducible .. we know exaclty what is going on in the vm box when the box was started up on vagrant up. Let me know what you think.